lms/djangoapps/experiments/tests/test_views.py

```diff
@@ -215,21 +215,21 @@ def test_permissions(self):
         """ Staff access is required for write operations. """
         url = reverse('api_experiments:v0:key_value-list')
 
+        user = UserFactory()
+        self.client.login(username=user.username, password=UserFactory._DEFAULT_PASSWORD)
+
         response = self.client.get(url)
         self.assertEqual(response.status_code, 200)
 
         response = self.client.post(url, {})
-        self.assertEqual(response.status_code, 401)
```

Comment:
Does this remove part of the test?  Maybe it wasn't needed, but was testing 401 and 403 condition, and now just 403.